x-pack/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -285,12 +335,37 @@ export function dataFrameAnalyticsRoutes({ router, mlLicense }: RouteInitializat
     mlLicense.fullLicenseAPIGuard(async (context, request, response) => {
       try {
         const { analyticsId } = request.params;
+
+        const { destinationIndex } = request.query;
+
+        // If user checks box to deleete the destinationIndex associcated with the job
```

Comment:
typo in deleete

Reply:
Fixed in https://github.com/elastic/kibana/commit/d14fafab86d64ef3afce4bda0f3c415ea85e672e